components/Footer/Footer.tsx

```diff
@@ -1,14 +1,20 @@
 import Image from 'next/image';
 import styles from './Footer.module.scss';
+import type FooterProps from './types';
```

Comment:
I like having types separated out like this, cool idea! I made the mistake of using `.tsx` for all my file extensions on the careers site project, you could use `.ts` for this instead and it would make more semantic sense.

Reply:
Alright, I've updated it to use the `.tsx` extension when appropriate.